src/mobu/services/business/notebookrunner.py

```diff
@@ -15,11 +15,15 @@
 from tempfile import TemporaryDirectory
 from typing import Any
 
+import yaml
 from httpx import AsyncClient
 from structlog.stdlib import BoundLogger
 
+from mobu.constants import GITHUB_REPO_CONFIG_PATH
+from mobu.models.repo import RepoConfig
```

Comment:
This code is inside mobu, so use relative imports here (`...constants` and `...models.repo`).